components/crowpanel_epaper/crowpanel_epaper.cpp

```diff
@@ -148,8 +170,9 @@ void CrowPanelEPaperBase::send_command_sequence_(const uint8_t* sequence) {
     // Check if this command has a delay parameter
     if (num_args & DELAY_FLAG) {
       // This is a delay command, skip the delay value
-      // The actual delay is handled by the state machine
-      i++;
+      // TODO: The delay should be handled by the state machine 
+      uint8_t delay_ms = sequence[i++];
+      delay(delay_ms);
```

Comment:
Strangely this change causes my display (4.2inch) to get flipped on the y axis. (with 180deg rotation)
Is this needed for your display?